### PR TITLE
Fix wording

### PR DESCRIPTION
**Pull request by LorenzoSapora**
Fixes several grammatical errors, as well as reformats the opening descriptive sentence.

## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Appropriate language icon(s) added if applicable
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by serhii-londar** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 1 link issue, a project collaborator will take care of these, thanks :)</td>
    </tr>
  </tbody>
</table>

#### Link issue by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
417 | [301](https://httpstatuses.com/301) | https://github.com/z-------------/cumulonimbus redirects to<br>https://github.com/z-------------/CPod


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by LorenzoSapora** [comment]
@serhii-londar This has failed the CI test, but only on the fact it has "multiple" changes within the file. I honestly don't think you'd want me to push a pull request for every typo in the README, so unless you have objections, I believe this is ready to be merged.

Open to comments, though.

File: README.md [hunk 1]
```diff
@@ -2,16 +2,16 @@
   <img src="./icons/icon.png">
 </p>
 
-# Awesome macOS open source applications.
```
Comment (serhii-londar) [inline README.md]: I don't think it should be capitalized.

**Comment by LorenzoSapora** [comment]
On reflection, I agree, "macOS" is used by Apple, over "MacOS". Changes will come shortly

**Comment by LorenzoSapora** [comment]
Done. Please review.

**Comment by serhii-londar** [comment]
@LorenzoSapora Thank you! 👍

**Pull request merged**